jetstream/tests/test_config.py

```diff
@@ -456,6 +458,43 @@ def test_segment_definitions(self, experiments):
         assert "{{" not in configured.experiment.segments[1].data_source.from_expr
         assert "2019-12-01" in configured.experiment.segments[1].data_source.from_expr
 
+    def test_pre_treatment_config(self, experiments):
+        config_str = dedent(
+            """
+            [metrics]
+            weekly = ["spam"]
+
+            [metrics.spam]
+            data_source = "main"
+            select_expression = "1"
+
+            [metrics.spam.statistics.bootstrap_mean]
+            num_samples = 10
+
+            [metrics.spam.statistics.bootstrap_mean.pre_treatments.remove_nulls]
+
+            [metrics.spam.statistics.bootstrap_mean.pre_treatments.log]
+            base = 20.0
+
+            [metrics.spam.statistics.bootstrap_mean.pre_treatments.censor_highest_values]
```

Comment:
I'm not sure if this is the most convenient way to specify pre-treatments. These specifiers get pretty long.

Reply:
yeah, hm, let's think about this